frontend/packages/enduser-frontend/src/localization/app/fi.json

```diff
@@ -1005,6 +1005,9 @@
         "Sallitut tiedostomuodot:",
         "PDF, JPEG/JPG, PNG ja DOC/DOCX"
       ]
-    }
+    },
+    "modal-header": "Käsittely kesken",
+    "modal-message": "Tiedosto ei ole juuri nyt ladattavissa. Kokeile hetken kuluttua uudelleen.",
+    "modal-confirm": "Sulje"
```

Comment:
Should probably ask for these at least in Swedish as well if you haven't already.

Reply:
Yep looking for these atm